source/css/scss/vendor_override/foundation/_buttons.scss

```diff
@@ -6,21 +6,19 @@
 button, a.button, .button {
 	font-size: $button-font-sm;
 	font-weight: $font-weight-bold;
-	height: $button-height-med - $button-height-med-reduction;
-	line-height: $button-height-med - $button-height-med-reduction;
```

Comment:
do we not need to define the height and line-height here because the default values are what we want?

Reply:
It's my understanding that the default values are good since I added the padding variables for the small and medium up situations. This will cause issues with other buttons throughout the site, but we've started a running list of those issues in the ticket.